dataproc/snippets/quickstart/quickstart_test.py

```diff
@@ -25,7 +25,7 @@
 
 
 PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
-REGION = "us-central1"
+REGION = "us-central2"
```

Comment:
```suggestion
REGION = "us-west1"
```
i think this one is more eco friendly 🌍